### ISSUE TITLE
Convert the python list to the numpy array for the python embedding at the base class.

### ISSUE THREAD

**Issue by hsoh-u** [dtcenter/MET#2386]
The python list and the numpy array are accepted for the python embedding. The current implementation gives a INFO message if the python list contains numpy data type members.
- no info message with 1) python list  & python general numeric data type members and 2) numpy array & numpy data type members
- info message with the python list & the numpy data type members
```
==INFO_PYTHON== Recommend using numpy instead of python list for obs_hgt (<class 'numpy.float32'>)
==INFO_PYTHON== Recommend using numpy instead of python list for obs_val (<class 'numpy.float32'>)
```

## Describe the Enhancement ##
MET codes handle better with the numpy array than the python list for the python embedding. It's better to convert the python list to numpy array instead of giving a INFO message at the base python class (met_point_obs.py).

### Time Estimate ###
*Estimate the amount of work required here.*
4 hours

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*
2702691

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [ ] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by DanielAdriaansen** [comment]
@hsoh-u was this required for #2443 or was this fixed in #2443 because you were in the code and easy to do at the same time?

**Comment by hsoh-u** [comment]
> @hsoh-u was this required for #2443 or was this fixed in #2443 because you were in the code and easy to do at the same time?

It was to do at the same time instead of creating tqo pull requests
